Patches/Kit's Gunpowder Weapons/KitsGunpowder_Guns.xml

```diff
@@ -72,7 +72,7 @@
 				<capacities>
 				<li>Stab</li>
 				</capacities>
-				<power>6</power>
+				<power>18</power>
```

Comment:
This seems to be using the spear stats. I plugged some stats from a brown bess bayonet (which is far narrower and slightly heavier, doing less damage and being slower), and it's a bit slower. I came up with the following:
Power: 14
Blunt AP: 2.16
Sharp AP: 1.44
Cooldown: 1.65

Your thoughts?

Reply:
the stat of bayonet is already worse than a spear, if you see the need to make it even worse then feel free to make the edit. I do not think it should be much worse considering that rifles do not get the melee bonus(when they really should).